src/algorithms/reco/TrackClusterMatch.cc

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2025 Tristan Protzman
+
```

Comment:
This algorithm can no work on older supported EDM4eic's:
https://github.com/eic/EICrecon/actions/runs/14066224254/job/39389282345#step:8:551
you'll need to wrap it all in `ifdef`.

```suggestion

#include <edm4eic/EDM4eicVersion.h>
#if EDM4EIC_VERSION_MAJOR >= 8
```